src/basic/log.c

```diff
@@ -1478,3 +1478,18 @@ void log_setup(void) {
         if (log_on_console() && show_color < 0)
                 log_show_color(true);
 }
+
+void log_dropped_errors(size_t num_dropped_errors, int level) {
+        if (num_dropped_errors == 0)
+                return;
+
+        log_full(level, "Dropped %zu similar message(s)", num_dropped_errors);
```

Comment:
this function should probably also reset the num_dropped_errors counter back to zero, instead of leaving that to the caller

Reply:
With the new approach (using the internal counter of the `RateLimit` object), this is not needed anymore